x/concentrated-liquidity/lp.go

```diff
@@ -428,12 +428,11 @@ func (k Keeper) isLockMature(ctx sdk.Context, underlyingLockId uint64) (bool, er
 		return false, err
 	}
 
-	// Check if the lock has expired
-	if underlyingLock.EndTime.After(ctx.BlockTime()) {
-		// Lock is still active, so we cannot withdraw from this position
+	if underlyingLock.EndTime.IsZero() {
+		// Lock is still active, so we can't withdraw from this position
 		return false, nil
 	}
 
-	// Lock has expired, so we can withdraw from this position
-	return true, nil
+	// Return if the lock has expired
+	return underlyingLock.EndTime.Before(ctx.BlockTime()), nil
```

Comment:
There is no case where this returns true, both return false. Please add one to cover both outcomes

Reply:
true https://github.com/osmosis-labs/osmosis/blob/69605cdc49d631233abaaba25c548995ee244c35/x/concentrated-liquidity/lp_test.go#L1146-L1150

false https://github.com/osmosis-labs/osmosis/blob/69605cdc49d631233abaaba25c548995ee244c35/x/concentrated-liquidity/lp_test.go#L1151-L1155